aws/rust-runtime/aws-types/src/endpoint.rs

```diff
@@ -4,6 +4,7 @@
  */
 
 //! AWS SDK endpoint support.
+#![allow(deprecated)]
```

Comment:
We expect to eventually remove the resolver fn right? If yes, we should create an issue and add a TODO here

Reply:
Actually, now that I think about it, perhaps we just need to create an issue that says "remove all deprecated code" that we pick up before GA